tests/PHPStan/Command/ReformatCommandTest.php

```diff
@@ -0,0 +1,88 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Command;
+
+use Nette\Utils\Json;
+use PHPStan\Analyser\Error;
+use PHPStan\Command\ErrorFormatter\JsonErrorFormatter;
+use PHPStan\Testing\TestCase;
+use Symfony\Component\Console\Tester\CommandTester;
+use PHPStan\Command\Output;
+
+class ReformatCommandTest extends TestCase
```

Comment:
Instead of this test, create a JsonErrorFormatterDeserializer and test it directly, instead of indirect testing through Commandtester. Thank you.